source/hardware/controllers/pioneer_ddj_sb.rst

```diff
@@ -15,8 +17,11 @@ Windows
 
 Windows Vista, Windows 7 and Windows 8 are supported. You can download the latest drivers and firmware from http://www.pioneerdj.com/en/support/software/ddj-sb.
 
-**IMPORTANT for Windows users**: If you are having issues getting both sound outputs to work properly, please try using a different Sound API (see the Preferences menu). DirectSound seems to have
-issues with this controller. WASAPI seems to work fine, as does MME, although only with higher latencies. On Windows 10, only MME works.
+.. note::
+   If you're using windows and have problems getting both sound outputs to work properly, please try using a different Sound API (see the Preferences menu).
```

Comment:
reference API?
Add a link to Pref > Sound hardware?

Reply:
Done.